content/blog/2016/2016-05-26-gsoc2016-automatic-plugin-documentation.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Automatic Plugin Documentation"
+tags:
+- jenkins
+- gsoc
+- plugins
+author: cynthia
+---
+=== About me 
+I am Cynthia Anyango from Nairobi , Kenya . I am a second yeat student at Maseno University . I am currently specializing on Ruby on Rails and trying to learn Python . I recently started contributing to Open source projects . My major contribution was at Mozilla , where I worked with the QA for Cloud services . I did manual and automated tests for various cloud services  . I wrote documentation too . Above that , I am competent and I am always passionate about what I get my hands on . 
```

Comment:
second yea*r* : typo

(Side note: A possible good practice you want to adopt when using markup languages (be it asciidoc, markdown if you're getting desperate, or whatever), is to try and separate sentences/or logical group of words by newlines. Though I agree it does not make a lot of sense from a pure litterature PoV, it can generally reduce the diffs when the text has to be amended, and makes it easier to comment things (here, we are commenting many things since that line above represents 6 lines for me e.g.))